app/views/pafs_core/projects/steps/financial_year_alternative.html.erb

```diff
@@ -13,7 +13,7 @@
             <p><%= t('.lede') %></p>
         
             <%= f.govuk_number_field :project_end_financial_year,
-              min: Time.now.uk_financial_year,
+              min: Date.today.uk_financial_year,
```

Comment:
I'd suggest being consistent and keep using `Time.zone.today.uk_financial_year` instead of `Date.today.uk_financial_year`